### PR TITLE
Bump v1.2.1.

### PR DESCRIPTION
**Pull request by n1k0**
r=? @leplatrem 

Note: I have no idea why the changelog has been generated with so many changes. This thing is getting out of control and should be dropped at some point, so we only handle changelog for releases on github.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,25 +1,44 @@
...
 
 **Fixed bugs:**
 
-- Broken links in the doc [\#293](https://github.com/Kinto/kinto.js/issues/293)
-- Fix JSM compatibility for Firefox adapter [\#274](https://github.com/Kinto/kinto.js/pull/274) ([leplatrem](https://github.com/leplatrem))
+- broken links in the doc [\#293](https://github.com/Kinto/kinto.js/issues/293)
+
+**Closed issues:**
+
+- Expose Chrome APIs [\#289](https://github.com/Kinto/kinto.js/issues/289)
```
Comment (leplatrem) [inline CHANGELOG.md]: I don't think we should mention closed issues if no associated PR were merged.

**Comment by leplatrem** [comment]
r+ to keep going forward. But definitely not fan of retroactive changes in changelog :)

**Comment by n1k0** [comment]
> But definitely not fan of retroactive changes in changelog :)

We'll drop that file anyway eventually. Unmaintainable.

**Comment by n1k0** [comment]
Dang! While performing the checks from the release procedure, I lately discovered that babel-polyfill should be a peer dependency:

```
➜  tmp node
> require("kinto")
Error: Cannot find module 'babel-polyfill'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/home/niko/tmp/node_modules/kinto/lib/index.js:39:3)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
```

I'm gonna have to release a 1.2.2 asap. GRMBL.

**Pull request merged**